raddb/mods-available/rest

```diff
@@ -141,7 +141,7 @@ rest {
 	#
 	#  WARNING: The server will not start if the server specified is unreachable.
 	#
-	connect_uri = "http://127.0.0.1:9090/"
+	connect_uri = "https://networkradius.com"
```

Comment:
We shouldn't point to our site in the default configs.  We will have thousands of people enable it and poke us.

I've just leave it as `127.0.0.1:143`, with a note that it needs to be changed.

Reply:
This is for rest not imap, so just, don't change it all...